docs/TROUBLESHOOTING.md

```diff
@@ -48,6 +48,18 @@ Once the SSH bastion has been setup, you can use either method to access the Win
   ```
 * You can now RDP into the Windows node at *localhost:2020* using an RDP client
 
+## Rebooting a Windows node
+
+In general, the operator minimize disruptions and avoid node reboots whenever possible. Certain operations and
+updates at the system level still require a traditional reboot process to ensure changes are applied correctly
+and securely.
+
+To reboot a Windows node, please follow the Openshift documentation on [Rebooting a node gracefully](https://docs.openshift.com/container-platform/latest/nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-gracefully_nodes-nodes-rebooting)
+to cordon, drain and reboot with the following command in PowerShell:
+```powershell
+  Restart-Computer -Force
+```
```

Comment:
The AWS issue+workaround should be listed in the same spot as these instructions IMO

Reply:
That was the first write-up. I moved apart so that the general node reboot process is documented and, as a side note, the limitation tracks the specific corner-case for AWS (which can be removed once resolved)

Reply:
Still I think the AWS specific information should be listed here. Manual node reboot is a troubleshooting step so that info doesn't need to be in the main README

Reply:
I see your point and agree. I followed the existing pattern by extending the list of limitations.

https://github.com/openshift/windows-machine-config-operator/blob/master/README.md#limitations

Reply:
> Still I think the AWS specific information should be listed here. Manual node reboot is a troubleshooting step so that info doesn't need to be in the main README

@saifshaikh48   Addressed in https://github.com/openshift/windows-machine-config-operator/pull/2291/commits/82055e05c19ac2ab05a55288e7be9a5dd86d8d82#diff-96160ca341eafe20466fcf7f74d2d720f3f05a614452c5ab41f32d8756a3fec0R63

PTAL